s3_prod_push.sh

```diff
@@ -14,7 +14,6 @@ EXCLUDES="s3_excludes"
 
 openssl aes-256-cbc -K $encrypted_e2db0eb08244_key -iv $encrypted_e2db0eb08244_iv -in secrets.tar.enc -out secrets.tar -d
 tar xvf secrets.tar
-
```

Comment:
What is the significance of this change? If it is just removing a blank line, this file does not need to be changed.